### DISCUSSION TITLE
Contact request on Discord

### DISCUSSION THREAD

**Discussion by mmm-it** [tacosontitan/tacosontitan#12] [General]
Hello I saw you have some more experience with the .NetUpgrade Tool than I do, so I tried to contact you on Discord but couldn't reach you yet. If you could pass in Discord for a little discussion I would greatly appreaciate that.
Thanks a lot
Max

**Comment by tacosontitan** [comment]
Can you clarify how you concluded that I have more experience with the .NET upgrade tool? I only respond to direct messages on Discord from sponsors, as noted in my profile here on GitHub:

![image](https://github.com/tacosontitan/tacosontitan/assets/65432314/ca6fd94b-b4cb-49ea-ac54-43730562d3cc)

**Comment by tacosontitan** [comment]
Since it's been two weeks with no response I'm going to close this discussion. Happy Holidays!

![MyDressUpDarlingSonoBisqueDollWaKoiWoSuruGIF](https://github.com/tacosontitan/tacosontitan/assets/65432314/05234af2-737f-4b2f-aec9-d4632a5767ab)
